### PR TITLE
Project metadata

### PR DESCRIPTION
**Pull request by malyvsen**
One more thing that I think might be good is to mock `~/.kale.toml` when doing tests - I needed that in `road-segmentation` just now, and it seems like we generally want projects to rely on `data-kale`.

### CODE REVIEW

File: {{cookiecutter.repository_name}}/.github/workflows/publish.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ jobs:
         python-version: '3.8'
 
     - name: Install dependencies
+      env:
+        POETRY_HTTP_BASIC_PACKAGR_USERNAME: irrelevant-when-using-token
```
Comment (samedii) [inline {{cookiecutter.repository_name}}/.github/workflows/publish.yml:20]: I personally like Jim's convention of using `__TOKEN__` as the username

Comment (samedii) [reply {{cookiecutter.repository_name}}/.github/workflows/publish.yml:20]: Are we certain that we want private repos to be a part of the template?

**Review by samedii — COMMENTED** [review]
Nice cleanup and fixes. I didn't add private before because I was a little uncertain of if we want it to be default? I also feel uncertain about kale as default, what do you think?
@malyvsen @Jim-Holmstroem @FelixAbrahamsson

**Comment by FelixAbrahamsson** [comment]
I was thinking that the templates could be something we showcase `pytorch-lantern` with to other people, so yeah I also don't think private packages should be part of it. I'm unsure about `data-kale`. Perhaps we need an "aiwizo" template and a "public" template?

**Comment by samedii** [comment]
We should at least not point to our private repository in the template. Might even be a minor security risk

I think both private repos and `data-kale` could be ok if we don't point to our stuff. However, private repo dependencies are quite rare, we don't have them in many cases so should probably not be default? We could still have it for publishing since people would likely only publish their model to a private repo and not to pypi?

**Review by samedii — APPROVED** [review]
Let's merge!

**Pull request merged**